signer/migrations/0003__create_tables.sql

```diff
@@ -319,3 +319,31 @@ CREATE TABLE sbtc_signer.swept_deposits (
 -- unique per 'sweep_transaction_id'.
 CREATE UNIQUE INDEX uix_swept_deposits_req_txid_req_output_index_pkgd_txid
     ON sbtc_signer.swept_deposits(deposit_request_txid, deposit_request_output_index, sweep_transaction_txid);
+
+
+CREATE TABLE sbtc_signer.bitcoin_txs_sighashes (
+    txid BYTEA PRIMARY KEY,
+    chain_tip BYTEA NOT NULL,
+    prevout_txid BYTEA NOT NULL,
+    prevout_output_index INTEGER NOT NULL,
+    sighash BYTEA NOT NULL,
+    prevout_type sbtc_signer.prevout_type NOT NULL,
```

Comment:
Do we need `prevout_type` here? We should be able to get it by joining to `bitcoin_tx_*` tables,  right?

Reply:
Hmmm, right now the `bitcoin_tx_*` tables only have confirmed inputs and outputs and I'm not sure if we should add this data there. @cylewitruk are you thinking that we should add this stuff there?

Reply:
@djordon I guess I was thinking that these prevouts should already be confirmed and stored in those tables when processing in the block observer? I've probably got my head spun around though :sweat_smile:

Reply:
Ohh, yeah I see. Hmm, I don't think deposits get added to those tables, just donations and sweep/sBTC transactions.

Just checked and we don't add deposits to those tables. But maybe we should?

Reply:
Yeah maybe we should for consistency, like to be able to say that those tables include all relevent inputs/outputs for all sbtc-relevant txs

Reply:
Note: Since we’ve removed the get query for this object, no instances of this struct are being created from the database. So, keeping or removing the field, without it being present in another table, won’t cause any issues for now. 

To keep things moving, I propose leaving the field for now (it's just an extra integer, so it shouldn’t cause any issues) and opening a separate issue to save all sBTC transactions in the block observer, what do you guys think?

Reply:
Issue opened https://github.com/stacks-network/sbtc/issues/931